python/looker_sdk/rtl/api_settings.py

```diff
@@ -122,6 +135,21 @@ def _bool(val: str) -> bool:
             raise TypeError
         return converted
 
+    def _override_settings(
```

Comment:
We can avoid this with some `cast`s, but I couldn't get anything else to infer the correct types without explicitly using the SettingsConfig keys.

Reply:
👍

Reply:
hm actually I see now there is some use of `cast` elsewhere. Personally I like the `cast` solution better, less code changes and personally I feel the code is more future-proof, this way we could easily miss keys we want.

I'll put up a version with that so you can compare